executor/aggfuncs/row_number.go

```diff
@@ -14,10 +14,17 @@
 package aggfuncs
 
 import (
+	"unsafe"
+
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/util/chunk"
 )
 
+const (
+	// DefPartialResult4RowNumberSize is the size of partialResult4AvgDecimal
```

Comment:
partialResult4AvgDecimal -> partialResult4RowNumber

Reply:
i will fix this.